### PR TITLE
Merging OZ 2.0.0.rc3

### PR DESCRIPTION
**Pull request by nventuro**
This brings over all changes from the latest (and hopefully last) OpenZeppelin release candidate, 2.0.0-rc3. There are some minor changes in contracts, and multiple test improvements (which can be safely ignored in oz-zos), but nothing that affects initializers changed. See [here](https://github.com/OpenZeppelin/openzeppelin-solidity/releases/tag/v2.0.0-rc.3) for the rc3 changelog.

### CODE REVIEW

File: contracts/token/ERC721/ERC721MetadataMintable.sol [hunk 1]
```diff
@@ -8,7 +9,11 @@ import "../../access/roles/MinterRole.sol";
  * @title ERC721MetadataMintable
  * @dev ERC721 minting logic with metadata
  */
-contract ERC721MetadataMintable is ERC721, ERC721Metadata, MinterRole {
+contract ERC721MetadataMintable is Initializable, ERC721, ERC721Metadata, MinterRole {
+  function initialize() public initializer {
```
Comment (spalladino) [inline contracts/token/ERC721/ERC721MetadataMintable.sol]: Shouldn't `ERC721.initialize()` be called here?

**Comment by spalladino** [comment]
Reviewed all commits comparing them to their oz-sol counterparts, and they all seem to be ported ok. Just left a question on an initializable.

File: contracts/token/ERC721/ERC721MetadataMintable.sol (hunk 1 above)
Comment (spalladino) [reply contracts/token/ERC721/ERC721MetadataMintable.sol]: Just found it was added in https://github.com/OpenZeppelin/openzeppelin-zos/pull/24/commits/26fe5a57907c741f370bfe02e5f164fe7d855b99. Let's remove commit fb1fb0913c3e213067977d29b1ed15dfd644ece0 from this PR and merge, and discuss initializers on #24.

Comment (nventuro) [reply contracts/token/ERC721/ERC721MetadataMintable.sol]: Merging since this and #24 together leave this as we need it be.

**Pull request merged**